zazu/util.py

```diff
@@ -236,3 +246,17 @@ def raise_uninstalled(pkg_name):
 
     """
     raise click.ClickException('{0} not found, install it via "apt-get install {0}" or "brew install {0}"'.format(pkg_name))
+
+
+def get_plugin_list(plugin_subclass):
+    """Helper function to pull lists of plugins
+ 
+    Args:
+        plugin_subclass (obj): the subclass of the plugin in question
+
+    Returns:
+       known_types (dict): dictionary of the known types of the plugin in question 
```

Comment:
dict: dictionary where keys are the string name of the plugin and values are from_config functions.